scripts/calculateRevenue/protocols/beefy/helpers.ts

```diff
@@ -44,11 +41,14 @@ export async function getNearestBlock(
   return blockTimestampData.height
 }
 
-// TODO: Memoize this function so it's not repeated for every user address
+export const getNearestBlock = memoize(_getNearestBlock, {
+  hash: (...params: Parameters<typeof _getNearestBlock>) => params.join(','),
```

Comment:
what's the default hash function?

Reply:
This is what I found: https://github.com/github/memoize/blob/main/index.ts#L2-L6.